source/core/JavaScriptRef.cpp

```diff
@@ -20,18 +20,18 @@ namespace Vireo {
 
 #if kVireoOS_emscripten
 extern "C" {
-    // JavaScript Reference methods to call from JS land
-    // Parameters: returnValue*, reference
-    extern void jsIsNotAJavaScriptRefnum(Boolean*, JavaScriptRefNum*);
+    extern void jsIsNotAJavaScriptRefnum(TypeRef, JavaScriptRefNum*, TypeRef, Boolean*);
 }
 #endif
 
 VIREO_FUNCTION_SIGNATURE2(IsNotAJavaScriptRefnum, JavaScriptRefNum, Boolean)
 {
+    TypeRef typeRefJavaScriptRefNum = TypeManagerScope::Current()->FindType("JavaScriptRefNum");
+    TypeRef typeRefIsNotARefNum = TypeManagerScope::Current()->FindType("Boolean");
     JavaScriptRefNum* refnumPtr = _ParamPointer(0);
     Boolean isNotARefnum = false;
     #if kVireoOS_emscripten
-        jsIsNotAJavaScriptRefnum(&isNotARefnum, refnumPtr);
+        jsIsNotAJavaScriptRefnum(typeRefJavaScriptRefNum, refnumPtr, typeRefIsNotARefNum, &isNotARefnum);
```

Comment:
These parameters were flipped from the instruction signature so made them consistent